java/java-tests/testSrc/com/intellij/psi/formatter/java/JavaFormatterAlignmentTest.java

```diff
@@ -49,6 +49,20 @@ public void testMethodAndChainedField() throws Exception {
     );
   }
 
+  public void testChainedMethodWithComments() throws Exception {
+    getSettings().ALIGN_MULTILINE_CHAINED_METHODS = true;
+    doMethodTest("AA.bb()\n" +
+                 ".cc() // comment after line\n" +
+                 ".dd()\n " +
+                 "        /* block comment on empty line */\n" +
```

Comment:
Think I get it :)

Block comments were also affecting alignment, so you've added it here. Am I right? If so, I think we could create two tests instead: one for one-line comment and another for block comment. What do you think about it?